### PR TITLE
Use viewbinding over manual findViewById

### PR DESCRIPTION
**Pull request by Animeshz**
Closes https://github.com/ayush0402/tic-tac-toe-foss/issues/2

(dupe of #22, due to force push)

### CODE REVIEW

**Review by ayush0402 — APPROVED** [review]
Changes LGTM. Just a nit suggestion, Rename the title to have the prefix `Fix #issuenumber` as well and also rename the PR description to contain `Fixes #issuenumber` just as a convention we are following.

File: app/src/main/java/com/ayush/tictactoe/MainActivity.kt [hunk 1]
```diff
@@ -7,138 +7,131 @@ import android.widget.Button
...
+    private lateinit var binding: ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
 
        val boardstatus = Array(3){IntArray(3) {-1} }
 
     val msgtext=findViewById<TextView>(R.id.msgtext)
```
Comment (Rohit1173) [inline app/src/main/java/com/ayush/tictactoe/MainActivity.kt:22]: enable viewBinding for msgtext

Comment (Rohit1173) [reply app/src/main/java/com/ayush/tictactoe/MainActivity.kt:22]: enable viewBinding for `resetbtn` too

**Comment by Rohit1173** [comment]
left some comments PTAL  @Animeshz , Thanks

File: app/src/main/java/com/ayush/tictactoe/MainActivity.kt (hunk 1 above)
Comment (Animeshz) [reply app/src/main/java/com/ayush/tictactoe/MainActivity.kt]: Oh yeah sorry, fixed!

Comment (Rohit1173) [reply app/src/main/java/com/ayush/tictactoe/MainActivity.kt]: Good @Animeshz and from the next time don't resolve conversations by yourself, the person who opens it has to mark it resolved.

**Comment by Rohit1173** [comment]
LGTM, thanks

**Pull request merged**